src/main/webapp/app/course/learning-paths/learning-path-graph/learning-path-graph.component.html

```diff
@@ -22,7 +22,13 @@
         <ng-template #nodeTemplate let-node>
             <svg:g class="node" [attr.width]="node.dimension.width" [attr.height]="node.dimension.height">
                 <svg:foreignObject [attr.width]="node.dimension.width" [attr.height]="node.dimension.height">
-                    <jhi-learning-path-graph-node class="node" [courseId]="courseId" [node]="node" (click)="nodeClicked.emit(node)"></jhi-learning-path-graph-node>
+                    <jhi-learning-path-graph-node
+                        class="node"
+                        [class.current]="highlightedNode && node.id === highlightedNode.id"
```

Comment:
```suggestion
                        [class.current]="node.id === highlightedNode?.id"
```
Does this work as well?